libr/core/cmd_search.c

```diff
@@ -2349,7 +2353,11 @@ static int cmd_search(void *data, const char *input) {
                 //fin = ini + s->size;
         }
  */
-	maxhits = r_config_get_i (core->config, "search.maxhits");
+	core->search->maxhits = r_config_get_i (core->config, "search.maxhits");
+	// TODO default search.maxhits to UT64_MAX
+	if (!core->search->maxhits) {
```

Comment:
If `search.maxhits` is zero (default), `core->search->maxhits` will be `UT64MAX`.